src/request/request-tab.js

```diff
@@ -34,7 +34,8 @@ module.exports = {
 // TODO: Need to come up with a better self registration process
 require('./components/request-detail-panel-execution');
 var request = require('./components/request-detail-panel-request');
-require('./components/request-detail-panel-logging');
+var LoggingComponent = require('./components/request-detail-panel-logging').Logging;
+var LoggingViewModel = require('./view-models/LoggingViewModel').LoggingViewModel;
```

Comment:
I get you point. I guess part of my feeling is that this are models for the "components" so if anything the folder should be "component-models" but that seems a little long winded. Also the fact that besides in testing, no other sub system besides the components use these models. Given this, if they sit as a peer, it almost feels like then its a peer of the rest of the sub systems which isn't really the case. Does that make any sense?